### PR TITLE
destination-postgres: Add tunnel heartbeats and keepalive

### LINKED ISSUE #33871: SSH bastion times out with destination-postgres V2
See slack thread https://airbytehq-team.slack.com/archives/C03C4AVJWG4/p1703011385519419

### PR DESCRIPTION
**Pull request by gisripa**
## What
Potentially..  Fixes: #33871 

## How
See CDK changes https://github.com/airbytehq/airbyte/pull/33727
New CDK changes - Upgrade sshd-mina to latest version, more interested in this fix https://github.com/apache/mina-sshd/blob/master/docs/changes/2.9.2.md#channel-windows

### CODE REVIEW

File: airbyte-integrations/connectors/destination-postgres-strict-encrypt/build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.8.0'
+    cdkVersionRequired = '0.8.1'
```
Comment (evantahler) [inline airbyte-integrations/connectors/destination-postgres-strict-encrypt/build.gradle:7]: shouldn't this be 0.11.1?

Comment (gisripa) [reply airbyte-integrations/connectors/destination-postgres-strict-encrypt/build.gradle:7]: yeah. derp. fixed it.

**Review by edgao — APPROVED** [review]

**Review by edgao — APPROVED** [review]
:yolo: (but please also tag our friends in db sources as fyi)

**Pull request merged**